build/01_Build.bat

```diff
@@ -15,6 +15,7 @@ xcopy "%~dp0/../src/NSwag.ConsoleCore/bin/release/netcoreapp2.1/publish" "%~dp0/
 xcopy "%~dp0/../src/NSwag.ConsoleCore/bin/release/netcoreapp2.2/publish" "%~dp0/../src/NSwag.Npm/bin/binaries/NetCore22" /E /I /y
 xcopy "%~dp0/../src/NSwag.ConsoleCore/bin/release/netcoreapp3.0/publish" "%~dp0/../src/NSwag.Npm/bin/binaries/NetCore30" /E /I /y
 xcopy "%~dp0/../src/NSwag.ConsoleCore/bin/release/netcoreapp3.1/publish" "%~dp0/../src/NSwag.Npm/bin/binaries/NetCore31" /E /I /y
+xcopy "%~dp0/../src/NSwag.ConsoleCore/bin/release/netcoreapp5.0/publish" "%~dp0/../src/NSwag.Npm/bin/binaries/Net50" /E /I /y
```

Comment:
Ah looks like a typo: netcoreapp5.0 should have been net5.0. See https://github.com/RicoSuter/NSwag/pull/3167

Reply:
ah